cosmos/infra/azuredeploy.json

```diff
@@ -253,6 +253,13 @@
                 "description": "Specifies a comma separated preferred region list."
             }
         },
+        "consistencyLevel": {
+            "type": "string",
+            "defaultValue": "SESSION",
+            "metadata": {

```

Comment:
We should ideally restrict the user to the following 5 values in the enum to prevent bad inputs:
{
  "properties": {
    "consistencyLevel": {
      "type": "string",
      "defaultValue": "SESSION",
      "enum": ["STRONG, BOUNDED_STALENESS, SESSION, CONSISTENT_PREFIX, EVENTUAL "]
    }
    ...
  }
}

Reply:
Great point! Looks good now?

Reply:
Looks good